inbm/cloudadapter-agent/fpm-template/usr/share/cloudadapter-agent/thingsboard/intel_manageability_batch_version_3.3.json

```diff
@@ -783,7 +878,7 @@
                 "col": 11
               },
               "21555927-43b5-f9d6-0174-bcd15bd1d96a": {
-                "sizeX": 4,
+                "sizeX": 5,
```

Comment:
Why is this sizeX changed? is this dashboard tested on thingsboard?

Reply:
> Why is this sizeX changed? is this dashboard tested on thingsboard?

Trigger Config Update button is bigger so assuming everyone views with 100% browser resolution so this has to be increased.

Reply:
This dashboard is tested on thingsboard, FYR http://172.30.249.216:8080/dashboards/5568ce10-f840-11ec-958c-1d503145ae62.